snapshot-replicator/monitoring.tf

```diff
@@ -0,0 +1,142 @@
+resource "aws_cloudwatch_metric_alarm" "lambda_rds_snapshot_copy_errors" {
+  count               = var.enable ? 1 : 0
+  alarm_name          = "rds_snapshot_copy_invocation_errors"
```

Comment:
The alarm name should contain `var.environment` so it doesn't collide with other setups